examples/nlp/text_classification/conf/text_classification_config.yaml

```diff
@@ -14,66 +12,73 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Config file for text classification with pretrained BERT models
+# Config file for text classification with pre-trained BERT models
 
 trainer:
-  gpus: 1 # the number of gpus, 0 for CPU
+  gpus: 1 # number of gpus (0 for CPU), or list of the GPUs to use e.g. [0, 1]
   num_nodes: 1
   max_epochs: 300
   max_steps: null # precedence over max_epochs
   accumulate_grad_batches: 1 # accumulates grads every k batches
   gradient_clip_val: 0.0
   amp_level: O0 # O1/O2 for mixed precision
-  precision: 16 # Should be set to 16 for O1 and O2, default is 16 as PT ignores it when am_level is O0
+  precision: 32 # Should be set to 16 for O1 and O2, ??default is 16 as PT ignores it when am_level is O0?? It is changing in new PT?
   distributed_backend: ddp
   row_log_interval: 1  # Interval of logging.
   val_check_interval: 1.0  # Set to 0.25 to check 4 times per epoch, or an int for number of iterations
   resume_from_checkpoint: null # The path to a checkpoint file to continue the training, restores the whole state including the epoch, step, LR schedulers, apex, etc.
+  num_sanity_val_steps: 0 # number of steps to perform validation steps for sanity check the validation process before starting the training, setting to 0 disables it
 
   checkpoint_callback: False  # Provided by exp_manager
   logger: False  # Provided by exp_manager
 
 model:
+  nemo_path: null # filename to save the model and associated artifacts to .nemo file
+  tokenizer:
+      tokenizer_name: ${model.language_model.pretrained_model_name} # or sentencepiece
+      vocab_file: null # path to vocab file 
+      tokenizer_model: null # only used if tokenizer is sentencepiece
+      special_tokens: null
+
+  language_model:
+    pretrained_model_name: bert-base-uncased
+    lm_checkpoint: null
+    config_file: null # json file, precedence over config
+    config: null 
+
+  classifier_head:
+    num_output_layers: 2
+    fc_dropout: 0.1
+
   dataset:
+    num_classes: ??? # The number of classes. 0 < Label <num_classes.
```

Comment:
This one is the only part we need to create a model. That is why it is mandatory. It also good to make the user set it and make them aware that they should set it.

Reply:
This num_classes is the only param need to be set to make a new model and it changes for every dataset. I think it is better to have it ??? so that user become aware of this param.